sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionAsyncApiCrudTest.java

```diff
@@ -154,6 +156,53 @@ public void queryItems() {
         }
     }
 
+    @Test(groups = {"encryption"}, timeOut = TIMEOUT)
+    public void queryItemsAggregate() {
+        long startTime = Instant.now().getEpochSecond();
+        List<String> actualIds = new ArrayList<>();
+        EncryptionPojo properties = getItem(UUID.randomUUID().toString());
+        cosmosEncryptionAsyncContainer.createItem(properties, new PartitionKey(properties.getMypk()),
+            new CosmosItemRequestOptions()).block();
+        actualIds.add(properties.getId());
+        properties = getItem(UUID.randomUUID().toString());
+        cosmosEncryptionAsyncContainer.createItem(properties, new PartitionKey(properties.getMypk()),
+            new CosmosItemRequestOptions()).block();
+        actualIds.add(properties.getId());
+        properties = getItem(UUID.randomUUID().toString());
+        cosmosEncryptionAsyncContainer.createItem(properties, new PartitionKey(properties.getMypk()),
+            new CosmosItemRequestOptions()).block();
+        actualIds.add(properties.getId());
+
+        // MAX query
+        String query1 = String.format("Select value max(c._ts) from c");
+        CosmosQueryRequestOptions cosmosQueryRequestOptions1 = new CosmosQueryRequestOptions();
+
+        SqlQuerySpec querySpec1 = new SqlQuerySpec(query1);
+        CosmosPagedFlux<JsonNode> feedResponseIterator1 =
```

Comment:
Can you please add tests for queries returning types like Integer, String etc

Reply:
I have added test cases for Integer, String.